bundles/org.smarthomej.binding.notificationsforfiretv/src/main/java/org/smarthomej/binding/notificationsforfiretv/internal/NotificationsForFireTVBindingConstants.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ * Copyright (c) 2021 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.notificationsforfiretv.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link NotificationsForFireTVBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Tom Blum - Initial contribution
+ */
+@NonNullByDefault
+public class NotificationsForFireTVBindingConstants {
+
+    private static final String BINDING_ID = "notificationsforfiretv";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_NOTIFICATION = new ThingTypeUID(BINDING_ID, "notification");
+    // List of all Parameters
+    public static final String TYPE = "type";
+    public static final String TITLE = "title";
+    public static final String MSG = "msg";
+    public static final String DURATION = "duration";
+    public static final String FONTSIZE = "fontsize";
+    public static final String POSITION = "position";
+    public static final String BKCOLOR = "bkgcolor";
+    public static final String TRANSPARENCY = "transparency";
+    public static final String OFFSET = "offset";
+    public static final String OFFSETY = "offsety";
```

Comment:
```suggestion
    public static final String OFFSET_Y = "offsetY";
```

Reply:
This is not possible because it represents the name of the part in a multipart. So the developer of Notificatins for Fire TV decided to define the name for the x offset only offset and the name for the y offset offsety. This is why i also defined all the x offset variables only offset and not offsetx.

Reply:
done